backend/src/main/java/com/handong/rebon/member/domain/Member.java

```diff
@@ -23,16 +25,27 @@
     @Embedded
     private Profile profile;
 
+    @Builder.Default
     @OneToMany(mappedBy = "member")
-    private List<Like> likes;
+    private List<Like> likes = new ArrayList<>();
 
+    @Builder.Default
     @OneToMany(mappedBy = "member")
-    private List<Empathy> empathies;
+    private List<Empathy> empathies = new ArrayList<>();
 
     public Member(Profile profile) {
         this.profile = profile;
     }
 
+    public boolean isSame(Member member) {
+        return id == member.getId();
+    }
```

Comment:
같은 클래스이므로 get 대신 .id로 비교하는게 좋아보여요


```suggestion
    public boolean isSame(Member member) {
        return id.equals(member.id);
    }
```